App.js

```diff
@@ -1,26 +1,43 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import React from 'react';
+import StepCounter from './Components/StepCounter';
```

Comment:
Let's remember about one line break between outer and local dependencies :)
Moreover, `/components` folder should be named in lowercase, capitalised rule goes only for the files